utils/python/CIME/case.py

```diff
@@ -1113,8 +1113,13 @@ def _check_testlists(self, compset_alias, grid_name, files):
 
         tests = Testlist(tests_spec_file, files)
         testlist = tests.get_tests(compset=compset_alias, grid=grid_name)
-        if len(testlist) > 0:
-            logger.info("\nThis compset and grid combination is not scientifically supported, however it is used in %d tests.\n"%(len(testlist)))
+        testcnt = 0
+        for test in testlist:
+            if test["category"] == "prealpha" or test["category"] == "prebeta" or "aux_" in test["category"]:
+                testcnt += 1
+        if testcnt > 0:
+            logger.info("\nThis compset and grid combination is not scientifically supported, however it is used in %d tests.\n"%(testcnt))
```

Comment:
expect now has an optional prefix argument that changes this from ERROR: to STOP:  as discussed in yesterdays meeting - I thought people objected to WARNING: